config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -579,6 +579,60 @@ presubmits:
           defaultMode: 256
           secretName: ssh-security
     trigger: (?m)^/test( | .* )pull-security-kubernetes-e2e-gce-alpha-features,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    cluster: security
+    context: pull-security-kubernetes-e2e-gce-csi-serial
+    labels:
+      preset-bazel-scratch-dir: "true"
+      preset-k8s-ssh: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
+      preset-service-account: "true"
+    name: pull-security-kubernetes-e2e-gce-csi-serial
+    rerun_command: /test pull-security-kubernetes-e2e-gce-csi-serial
+    spec:
+      containers:
+      - args:
+        - --ssh=/etc/ssh-security/ssh-security
+        - --root=/go/src
+        - --repo=github.com/kubernetes-security/kubernetes=$(PULL_REFS)
+        - --repo=k8s.io/release
+        - --upload=gs://kubernetes-security-prow/pr-logs
+        - --timeout=90
+        - --scenario=kubernetes_e2e
+        - --
+        - --build=bazel
+        - --cluster=
+        - --extract=local
+        - --gcp-node-image=gci
+        - --gcp-zone=us-central1-f
+        - --provider=gce
+        - --runtime-config=batch/v2alpha1=true,admissionregistration.k8s.io/v1alpha1=true
+        - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-csi-serial
+        - --test_args=--ginkgo.focus=CSI.*\[Serial\] --ginkgo.skip=\[Flaky\]|\[Feature:.+\]|gcePD-external
```

Comment:
this looks like it might run all Serial CSI tests, should we specifically focus on `GCE`?, I think we also want `[Disruptive]`

Reply:
All serial is fine I think.  Do we actually have jobs that are disruptive but not serial?

Reply:
`updateCSIDriver` one used to be - I thought it was fine since `Disruptive` also implied `Serial`. However, I am not aware of any since that was removed.

Reply:
Added disruptive tag to the focus